### ISSUE TITLE
Streamer indicated an error 2

### ISSUE THREAD

**Issue by sxarthur** [justfoolingaround/animdl#309]
[stream.py:224 - 2024-05-11 00:09:48,929 - streamer: WARNING] Streamer indicated an error 2.
Retry streaming this episode? [y/N]: y
      Currently playing: 'Episode 1'. 0 episodes in queue.
         This stream may be share-able [embeddable and playable as a single URL] (please note that subtitles, chapters
and titles won't be embedded).
         Share-ables: embed, direct url
[stream.py:224 - 2024-05-11 00:09:49,709 - streamer: WARNING] Streamer indicated an error 2.
Retry streaming this episode? [y/N]: y

don't know what to do

**Comment by Aeropro4** [comment]
Yeah and idm says: "The server replies that you don't have permissions to download this file.
Details: HTTP/I.1 403 Forbidden"
also it is happening in movies.
but ig its not gonna be fixed
